docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc

```diff
@@ -192,7 +184,7 @@ openstack overcloud node provision
   $PWD/network/baremetal_deployment.yaml
```

Comment:
Re: Run the `overcloud node provision` command...
Can we specify the action that the user is taking? E.g., “Configure the new network on the bare metal nodes”?

Reply:
Yes. As done for mons (even though I don't see the tripleo prerequisite section anymore, so I can't link it here), we can add the suggested step title (`Configure the new / missing network on the bare metal nodes`) and move this section at the very beginning of the adoption guide.

Reply:
basically I mean what we're discussing in comment https://github.com/openstack-k8s-operators/data-plane-adoption/pull/611#discussion_r1765538370